### PR TITLE
[RELENG-4589] change runner naming

### PR DESCRIPTION
**Pull request by erwan-b**

### CODE REVIEW

File: runners_manager/runner/RunnerManager.py [hunk 1]
```diff
@@ -107,10 +112,11 @@ def delete_runner(self, runner: Runner):
...
 
-    def next_runner_name(self):
-        name = f'{self.runner_counter}'
-        self.runner_counter += 1
-        return name
+    def generate_runner_name(self, vm_type: VmType):
+        return self.runner_name_format.format(index=self.runner_counter,
+                                              organization=self.github_organization,
+                                              flavor=vm_type.flavor,
+                                              image=vm_type.image).replace(' ', '-')
```
Comment (tcarmet) [inline runners_manager/runner/RunnerManager.py:119]: Not sure what case you encountered to need this replace, or if it's just a safety.

**Review by tcarmet — APPROVED** [review]
LGTM

File: runners_manager/runner/RunnerManager.py (hunk 1 above)
Comment (erwan-b) [reply runners_manager/runner/RunnerManager.py:119]: The name we use for images contain whitespaces :/

Comment (tcarmet) [reply runners_manager/runner/RunnerManager.py:119]: Are you talking about the name of the image that we give on the github side? or the image in openstack?

Meaning:
* On github I thought we called the image `centos7` or `centos` or `centos-7` -> Where I see no space here
* on openstack we don't have much control there and indeed I think some contain whitespace, but those are not the image name that we put on the runner name AFAIK

**Review by tcarmet — APPROVED** [review]
reapproving this PR as the discussion we had has been applied

**Pull request merged**